source/default_mode/DataNodes.h

```diff
@@ -66,6 +66,20 @@ emp::DataFile & SymWorld::SetupSymIntValFile(const std::string & filename) {
  */
 emp::DataFile & SymWorld::SetupHostIntValFile(const std::string & filename) {
   auto & file = SetupFile(filename);
+  AddHeadersToHostFile(file);
+  file.PrintHeaderKeys();
+  return file;
+}
+
+
+/**
+ * Input: The Empirical DataFile object tracking data nodes.
+ *
+ * Output: None.
+ *
+ * Purpose: To add host data nodes to be tracked to the host data file.
+ */
+void SymWorld::AddHeadersToHostFile(emp::DataFile & file){

```

Comment:
I like the existence of this kind of a method, I'm thinking about what the best name would be though, since it is doing more than adding the headers, and it's kind of not doing the headers, since that's the PrintHeaderKeys that isn't in this specifically. It's more setting up the columns right?